Unity Example/Assets/Discord RPC/Scripts/Presence/DiscordPresence.cs

```diff
@@ -151,6 +184,15 @@ public DiscordRPC.RichPresence ToRichPresence()
 			if (endTime.IsValid()) presence.Timestamps.End = endTime.GetDateTime();
 		}
 
+		if ((firstButton != null && !firstButton.IsEmpty()) || (secondButton != null && !secondButton.IsEmpty()))
+		{
+			presence.Buttons = new DiscordRPC.Button[]
+			{
+				new DiscordRPC.Button { Label = firstButton.label, Url = firstButton.url},
+				new DiscordRPC.Button { Label = secondButton.label, Url = secondButton.url}
+			};
+		}
```

Comment:
This will have to change for this PR to be accepted.
The buttons are an array, and thus should be listed in the inspector as an array. 

If discord only accepts 2 buttons, then this should reflect the base library and not be custom implemented in the unity package.

Reply:
fixed